src/main/java/org/jabref/gui/groups/GroupTreeView.java

```diff
@@ -570,7 +599,7 @@ private ContextMenu createContextMenuForGroup(GroupNodeViewModel group) {
     }
 
     private void addNewGroup() {
-        viewModel.addNewGroupToRoot();
```

Comment:
Please think of two options:

a) ENUM covering the options
b) two method (this is favored at [Java by Comparison](https://java.by-comparison.com/), item "Split Method with Boolean Parameters")